config/schemas/config.yaml

```diff
@@ -1473,6 +1473,99 @@ properties:
             title: Enable Kafka
             type: boolean
         additionalProperties: false
+      extraRoles:
+        title: Configure extra Roles for application developers
```

Comment:
Prefer to keep the title short like "Extra Roles", as that will be generated as "Extra Roles Schema" in the config, or "Extra Application Developer Roles" if you want to retain that in the title. It is fine with me if that is in the description instead.

Similar for the other titles.

Reply:
Like this https://github.com/elastisys/compliantkubernetes-apps/pull/2367/commits/6ccbd8e6dd8136858ffcaa6d0009853c7aee0372?

Reply:
Looks good, well done :+1: